packages/api-reference/src/components/ApiClientModal.vue

```diff
@@ -68,7 +68,6 @@ export { useApiClientStore } from '@scalar/api-client'
         </template>
         <ApiClient
           :proxyUrl="proxyUrl"
-          readOnly
```

Comment:
When using the API Client inside the reference, we want it to be editable (at least the address).